integration-tests/http/__tests__/draft-order/admin/draft-order.spec.ts

```diff
@@ -248,6 +336,81 @@ medusaIntegrationTestRunner({
         expect(response.status).toBe(200)
         expect(response.data.order.status).toBe("pending")
       })
+
+      it("should create reservations on draft order to order conversion", async () => {
+        await api.post(
+          `/admin/draft-orders/${testDraftOrder.id}/edit`,
+          {},
+          adminHeaders
+        )
+
+        await api.post(
+          `/admin/draft-orders/${testDraftOrder.id}/edit/items`,
+          {
+            items: [
+              {
+                variant_id: product.variants.find((v) => v.title === "L shirt")
+                  .id,
+                quantity: 1,
+              },
+            ],
+          },
+          adminHeaders
+        )
+
+        await api.post(
+          `/admin/draft-orders/${testDraftOrder.id}/edit/items`,
+          {
+            items: [
+              {
+                variant_id: product.variants.find((v) => v.title === "M shirt")
+                  .id,
+                quantity: 1,
+              },
+            ],
+          },
+          adminHeaders
+        )
+
+        let reservations = (await api.get(`/admin/reservations`, adminHeaders))
```

Comment:
this `reservations` are never used, they are refetched after calling `/confirm`. Did we want to check that the length here is 0 as well?